tests/test_parsers.py

```diff
@@ -52,7 +52,7 @@ def setUp(self) -> None:
 
         self.alignmentxml_file = f"{test_data_dir}/oaei-ordo-hp.rdf"
         self.alignmentxml = minidom.parse(self.alignmentxml_file)
-        self.metadata, self.curie_map = get_default_metadata()
+        self.curie_map, self.metadata = get_default_metadata()
```

Comment:
@hrshdhgd good catch - these were mixed up in so many places that I ended up replacing it with a named tuple to try and standardize it

Reply:
Wouldnt this be wrong here then? Doesnt get_default_metadata() return a `Metadata` object?

Reply:
A Metadata object is a namedtuple, therefore you can directly unpack it if you want